circuitpython_build_tools/build.py

```diff
@@ -36,6 +36,25 @@
 import subprocess
 import tempfile
 
+# pyproject.toml `py_modules` values that are incorrect. These should all have PRs filed!
+# and should be removed when the fixed version is incorporated in its respective bundle.
+
+pyproject_py_modules_blacklist = set((
```

Comment:
Did you mean `blocklist`?

Reply:
I would call it `deny_list` instead of `blackllist`